src/main/java/com/parsing/schedulers/LotIdScheduler.java

```diff
@@ -8,22 +11,39 @@
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
+import java.time.OffsetDateTime;
+import java.time.ZonedDateTime;
+import java.util.List;
+
 @Service
 @Slf4j
 @RequiredArgsConstructor
 @EnableAsync
 @EnableScheduling
 public class LotIdScheduler {
 
-    private static final long ONE_HOUR = 3_600_000L;
+    private static final long ONE_HOUR = 6_000_000;
 
     private static final long UPDATE_TIME = 36_000_000L;
 
+    private final LotIdRepository lotIdRepository;
+
+    private final LotIdExtractor lotIdExtractor;
+
     @Async
     @Scheduled(initialDelay = ONE_HOUR, fixedDelay = UPDATE_TIME)
```

Comment:
Could you initialDelay and fixedDelay add from yaml file? 
This will allow for flexible modification of these data while the function is running.

in yaml it could look like -> initialDelay: ${INITIAL_DELAY:6_000_000}
where INITIAL_DELAY - environment variable, 6_000_000 - default value if envirament variable not exist